PowerEditor/installer/nativeLang/estonian.xml

```diff
@@ -1,903 +1,1008 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-*Estonian language file for Notepad++*
-
-	** Created By:- Yusuf Mullaji *
-		Email Id-> yusufmullaji@rocketmail.com
-
+Estonian translation by Andres Traks
+https://github.com/AndresTraks/
```

Comment:
Atleast add my name in this translation. I have done this translation first. I know i used automatic translation but atleast i have contributed. @AndresTraks sorry for the incovenience but i have also tried to contribute. I made mistakes in translation but now you have corrected it so why my name should be removed?.  I am the first person to translate this so how can you say that, that  my name should not be considered.